src/pane-container.coffee

```diff
@@ -164,6 +164,17 @@ class PaneContainer extends Model
     else
       false
 
+  moveActiveItemToPane: (destPane) ->
+    item = @activePane.getActiveItem()
+    @activePane.moveItemToPane(item, destPane)
+    destPane.setActiveItem(item)
+    destPane.focus()
```

Comment:
I think it might be better to perform these `focus` calls in the view layer, for two reasons.

First, focus is a view-level concept. That method is really intended to be called from the view layer currently, leaving `activate` as the corresponding model level concept. The difference between them is pretty subtle: a pane can be "active" without being focused. We usually activate the most recently focused pane.

Second: It might be nice to keep these model-level methods a bit more focused on what they claim to do in their names, which is moving/copying items. I can even see them graduating to the public API someday, in which case messing with focus state might be surprising.

Let me know what you think of my reasoning here... This may have some impact on your tests, which are looking pretty good.